app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -319,9 +323,17 @@ public class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPage
     }
 
     override fun showRouteComplete() {
-        findViewById(R.id.footer_wrapper)?.visibility = View.GONE
         findViewById(R.id.resume)?.visibility = View.GONE
+        setCurrentInstruction(pager?.adapter?.count?.minus(1) ?: 0)
         notificationCreator?.killNotification()
+        (findViewById(R.id.destination_distance) as DistanceView).distanceInMeters = 0
```

Comment:
Why not create a reference to this so the code is more concise? It seems like we reference this id more than once